src/js/tech/tech.js

```diff
@@ -437,6 +437,15 @@ class Tech extends Component {
     return '';
   }
 
+  static isTech(component) {
+    /* jshint proto:true */
+    let proto = component.__proto__;
+    /* jshint proto:false */
+    if (Object.getPrototypeOf) {
```

Comment:
That won't help us since `component` in this case is a class itself, so, `component.constructor` will point at `Function`.
However, looks like babel puts sets `__proto__`, so, it might not be completely horrible to use it.